{
  "dataType": "CVE_RECORD",
  "containers": {
    "adp": [
      {
        "title": "CVE Program Container",
        "references": [
          {
            "url": "https://www.ibm.com/support/pages/node/7155078",
            "tags": [
              "vendor-advisory",
              "x_transferred"
            ]
          },
          {
            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/265264",
            "tags": [
              "vdb-entry",
              "x_transferred"
            ]
          }
        ],
        "providerMetadata": {
          "orgId": "af854a3a-2127-422b-91ae-364da2661108",
          "shortName": "CVE",
          "dateUpdated": "2024-08-02T19:16:49.464Z"
        }
      },
      {
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2023-42005",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "total"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-05-29T15:31:04.424637Z"
              }
            }
          }
        ],
        "providerMetadata": {
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "shortName": "CISA-ADP",
          "dateUpdated": "2024-05-29T15:33:24.524Z"
        },
        "title": "CISA ADP Vulnrichment"
      }
    ],
    "cna": {
      "title": "IBM Db2 on Cloud Pak for Data privilege escalation",
      "source": {
        "discovery": "UNKNOWN"
      },
      "metrics": [
        {
          "format": "CVSS",
          "cvssV3_1": {
            "scope": "UNCHANGED",
            "version": "3.1",
            "baseScore": 7.4,
            "attackVector": "LOCAL",
            "baseSeverity": "HIGH",
            "vectorString": "CVSS:3.1/AV:L/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H",
            "integrityImpact": "HIGH",
            "userInteraction": "NONE",
            "attackComplexity": "HIGH",
            "availabilityImpact": "HIGH",
            "privilegesRequired": "NONE",
            "confidentialityImpact": "HIGH"
          },
          "scenarios": [
            {
              "lang": "en",
              "value": "GENERAL"
            }
          ]
        }
      ],
      "affected": [
        {
          "cpes": [
            "cpe:2.3:a:ibm:db2:3.5:refresh_10:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:db2:4.0:refresh_9:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:db2:4.5:refresh_3:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:db2:4.6:refresh_6:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:db2:4.7:refresh_4:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:db2:4.8:refresh_4:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:db2_warehouse:3.5:refresh_10:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:db2_warehouse:4.0:refresh_9:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:db2_warehouse:4.5:refresh_3:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:db2_warehouse:4.6:refresh_6:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:db2_warehouse:4.7:refresh_4:*:*:*:*:*:*",
            "cpe:2.3:a:ibm:db2_warehouse:4.8:refresh_4:*:*:*:*:*:*"
          ],
          "vendor": "IBM",
          "product": "Db2 on Cloud Pak for Data",
          "versions": [
            {
              "status": "affected",
              "version": "3.5, 4.0, 4.5, 4.6, 4.7, 4.8"
            }
          ],
          "defaultStatus": "unaffected"
        }
      ],
      "references": [
        {
          "url": "https://www.ibm.com/support/pages/node/7155078",
          "tags": [
            "vendor-advisory"
          ]
        },
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/265264",
          "tags": [
            "vdb-entry"
          ]
        }
      ],
      "x_generator": {
        "engine": "Vulnogram 0.2.0"
      },
      "descriptions": [
        {
          "lang": "en",
          "value": "IBM Db2 on Cloud Pak for Data and Db2 Warehouse on Cloud Pak for Data 3.5, 4.0, 4.5, 4.6, 4.7, and 4.8 could allow a user with access to the Kubernetes pod, to make system calls compromising the security of containers. IBM X-Force ID: 265264.",
          "supportingMedia": [
            {
              "type": "text/html",
              "value": "\n\n<span style=\"background-color: rgb(255, 255, 255);\">IBM Db2 on Cloud Pak for Data and Db2 Warehouse on Cloud Pak for Data 3.5, 4.0, 4.5, 4.6, 4.7, and 4.8 could allow a user with access to the Kubernetes pod, to make system calls compromising the security of containers. IBM X-Force ID: 265264.</span>\n\n",
              "base64": false
            }
          ]
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "CWE",
              "cweId": "CWE-264",
              "description": "CWE-264 Permissions, Privileges, Access Controls"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "shortName": "ibm",
        "dateUpdated": "2024-05-29T12:53:04.315Z"
      }
    }
  },
  "cveMetadata": {
    "cveId": "CVE-2023-42005",
    "state": "PUBLISHED",
    "dateUpdated": "2024-08-02T19:16:49.464Z",
    "dateReserved": "2023-09-06T19:32:50.696Z",
    "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
    "datePublished": "2024-05-29T12:53:04.315Z",
    "assignerShortName": "ibm"
  },
  "dataVersion": "5.1"
}